src/database/migrations/uupdate-user-roles.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-unused-vars */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("tbl_users", {
```

Comment:
Man, why did you create second migration having the same table names?; please you may delete one and remain with only updated migration. and remember to check the migration file name